skfda/misc/hat_matrix.py

```diff
@@ -335,24 +335,24 @@ class KNeighborsHatMatrix(HatMatrix):
     regression algorithms, as explained below.
 
     .. math::
-        \hat{H}_{i,j} = \frac{K\left(\frac{d(e_j-e_i')}{h}\right)}{\sum_{k=1}^{
-        n}K\left(\frac{d(e_k-e_i')}{h_{ik}}\right)}
+        \hat{H}_{i,j} = \frac{K\left(\frac{d(x_j-x_i')}{h_i}\right)}
+        {\sum_{k=1}^{n}K\left(\frac{d(x_k-x_i')}{h_{i}}\right)}
 
-    For smoothing, :math:`e_i` are the points of discretisation
-    and :math:`e'_i` are the points for which it is desired to estimate the
-    smoothed value. The distance :math:`d` is the absolute value.
+    For smoothing, :math:`\{x_1, ..., x_n\}` are the points with known value
+    and :math:`\{x_1', ..., x_m'\}` are the points for which it is desired to
+    estimate the smoothed value. The distance :math:`d` is the absolute value.
 
-    For regression, :math:`e_i` are the functional data and :math:`e_i'`
-    are the functions for which it is desired to estimate the scalar value.
-    Here, :math:`d` is some functional distance.
+    For regression, :math:`\{x_1, ..., x_n\}` are the functional data and
+    :math:`\{x_1', ..., x_m'\}` are the functions for which it is desired to
+    estimate the scalar value. Here, :math:`d` is some functional distance.
 
     In both cases, :math:`K(\cdot)` is a kernel function and
-    :math:`h_{ik}` is calculated as the distance from :math:`e_i'` to it's
-    :math:`k`-th nearest neighbour in :math:`\{e_1, ..., e_n\}`
+    :math:`h_{i}` is calculated as the distance from :math:`x_i'` to its
+    :math:`n\_neighbors`-th nearest neighbor in :math:`\{x_1, ..., x_n\}`
```

Comment:
Don't use `:math:` for `n_neighbors`. use a double backtick (\`\`n_neighbors``) instead, to use a monospaced font.